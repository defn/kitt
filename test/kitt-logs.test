#!/bin/bash

function main {
  kitt logs --help
}

main "$@"
