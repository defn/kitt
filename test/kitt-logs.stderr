USAGE: kitt logs [flags] --  args

Calls docker-compose logs to tail service logs

args: arguments to docker-compose

flags:
  -h,--help:  show this help (default: false)
